crates/wash-cli/tests/wash_reg.rs

```diff
@@ -69,7 +63,7 @@ fn integration_reg_pull_comprehensive() {
             "--destination",
             comprehensive_echo.to_str().unwrap(),
             "--digest",
-            "sha256:a17a163afa8447622055deb049587641a9e23243a6cc4411eb33bd4267214cf3",
+            "sha256:079275a324c0fcd0c201878f0c158120c4984472215ec3f64eb91ba9ee139f72",
```

Comment:
Did we want to move this SHA out too? Also, I wonder if they also make sense in `common`